guides/release/components-update/component-arguments-and-html-attributes.md

```diff
@@ -0,0 +1,154 @@
+If you want to _reuse_ a component in multiple places, you'll need a way to
```

Comment:
Might it make sense to add a sentence or two about the motivation behind making components reusable or is this something that will already be discussed in another part of the Octane Component Guides? If not, what do you think about something along the lines of

```suggestion
Components become useful building blocks of our app if we make them _reusable_. When we reuse components efficiently, we can avoid having to recreate parts of our app again and again. If you want to _reuse_ a component in multiple places, you'll need a way to
```

Reply:
Thanks! Incorporated!